main.html

```diff
@@ -5,50 +5,53 @@
-                <p>Contact</p>
-            </div>
+    <div class="container">
+    <header class="my-header">
+        <div class="time">
+            <img src="mobile_menu_images/Bars/Status-Bars/Time-Style.jpg" alt="Header image">
         </div>
-        <div class="logo-placeholder">
-            <p>My Logo</p>
+        <div class="status-bar">
+            <img src="mobile_menu_images/Bars/Status-Bars/Group3.jpg" alt="Bar Status icons">
         </div>
-    </div>
+    </header>
     
-        <div class="social-media-icons">
-            <img src="images/social-media-icons.jpg">
+    <div class="rectangle"><img src="mobile_menu_images/Bars/Status-Bars/Rectangle.jpg" alt="Divider"></div>
+
+    <nav class="main-menu">
+      <div class="menu-container">
+        <div class="logo-frame">
+            <p>SIKO NORBERT</p>
         </div>
-        <div class="contact-form-background-shapes">
-            <img src="images/contact-form-background-shapes@2x.jpg">
+        <img src="mobile_menu_images/Normal-Button/Tertiary/Enabled.jpg" alt="Button">
+      </div>
+    </nav>
+
+    <section class="headline">
+        <div class="first-text">
+            <!---  I used without tag because the tag put automatically the margin  --->
+            <h1>I’m Norbert. <br>Glad to see you!</h1>
         </div>
-        <div class="header-bg">
-            <img src="images/Header-bg@2x.jpg">
+        <div class="second-text">
+            <!---  I used without tag because the tag put automatically the margin  --->
+        <h1>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h1>
         </div>
-
-        <div class="header">
-        <div class="text">
-            <h1>I’m Norbi
-                Glad to see you!</h1>
-            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <div class="social-media">
-                <div class="supporting-text">
-                    <p class="lets-connect">LET’S CONNECT</p>
-                </div>
-            </div>
-        </div>   
+        <div class="social-media">
+             <!---  I used without tag because the tag put automatically the margin  --->
+            LET'S CONNECT
         </div>
+        <div class="social-media-icons">
+            <a><img src="mobile_menu_images/Twitter.jpg" alt="Twitter icon"></a>
+            <a><img src="mobile_menu_images/Linkedin-icon.jpg" alt="Linkedin icon"></a>
+            <a><img src="mobile_menu_images/Metaverse.jpg" alt="Metaverse icon"></a>
+            <a><img src="mobile_menu_images/Github.jpg" alt="Github icon"></a>
+            <a><img src="mobile_menu_images/Vector.jpg" alt="Thumb up icon"></a>
+        </div>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon should also have a link (`a` tag). Kindly apply this change to your code.
- [ ] Please, ensure that your icons are aligned at the same line to fit the design. 
1. **your design**
![image](https://user-images.githubusercontent.com/80612925/170720075-fabc4968-604b-4070-b413-55cdd48ca193.png)

3. **the figma design**
![image](https://user-images.githubusercontent.com/80612925/170719964-a5a7d01b-f8ea-4b76-8b42-5b65b6eaa789.png)